appscale/tools/appscale_tools.py

```diff
@@ -818,6 +818,15 @@ def terminate_instances(cls, options):
     except (IOError, AppScaleException):
       # Don't fail if we cannot find the configuration.
       pass
+    except AppControllerException as e:
+      # If we are terminating a cloud infrastructure, we should log and
+      # continue.
+      if (infrastructure in InfrastructureAgentFactory.VALID_AGENTS and
+            options.terminate):
+        AppScaleLogger.warn(e)
```

Comment:
@whoarethebritons , is this error ignored for cloud mode because instances will be terminated (removed) later for cloud (so AppScale shouldn't necessarily be properly terminated there)?
Maybe then it makes sense to skip `RemoteHelper.terminate_virtualized_cluster` for cloud if terminate flag is set?